docs/squeezing_avr.md

```diff
@@ -189,6 +189,7 @@ That said, there are a number of Pro Micro replacements with ARM controllers:
 * [Proton C](https://qmk.fm/proton-c/) (out of stock)
 * [Bonsai C](https://github.com/customMK/Bonsai-C) (Open Source, DIY/PCBA)
 * [Raspberry Pi 2040](https://www.sparkfun.com/products/18288) (not currently supported, no ETA)
```

Comment:
```suggestion
* [Adafruit KB2040](https://learn.adafruit.com/adafruit-kb2040)
* [SparkFun Pro Micro - RP2040](https://www.sparkfun.com/products/18288)
```

These should be updated together with Blok as all 3 RP2040 boards will be supported in the coming merge cycle.

Reply:
Fully agree.

Reply:
> ```suggestion
> * [STeMCell](https://github.com/megamind4089/STeMCell) (Open Source, DIY/PCBA)
> * [Adafruit KB2040](https://learn.adafruit.com/adafruit-kb2040)
> * [SparkFun Pro Micro - RP2040](https://www.sparkfun.com/products/18288)
> ```
> 
> These should be updated together with Blok as all 3 RP2040 boards and the STM controller will be supported in the August merge cycle.

Nice, thanks! 
Okay, I'm still getting the hang of all this, so correct me if I'm wrong; but if I commit your edit to my PR;
- this is just updating/"replacing" the _old_ "RasPi 2040 (not supported, ... )"-entry by removing the incompatibility warning, as well as adding those additional 2 devboards to the list?
- but everything originally listed before-&-after this edit—including the Blok addition—will be kept? 

Basically, it appears to me that your edit was you just taking this opportunity to further-update _another_ entry on the list, _on-top of_ my original Docs addition, in-order to keep it all together in one PR commit - and I just want to make sure I'm understanding this correctly before I go ahead, so I can better understand this aspect of contributing & PR's. 🙃

Reply:
Your understanding is correct. The three green lines are showing what will be added if you commit the suggested changes. Your blok addition will remain unchanged below them.

Reply:
Boardsource just announced that they will start shipping so the prefix `(currently in Pre-Order phase, estimated In-Stock "August 2022")` should be removed now.

Reply:
Gotcha, will do & thanks!